compiler/version.json

```diff
@@ -0,0 +1,3 @@
+{
+  "package-version": "3.0.0.dev"
```

Comment:
Shall we start at 2.9.0 while we are testing, then bump to 3.0.0 next week after we have gone through the package publishing process once with the new versions and names?

Reply:
Done :)